app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -74,18 +74,35 @@ class SupplementalApplicationPage extends React.Component {
   }
 
   handleSavePreference = async (preferenceIndex, application) => {
```

Comment:
@fddayan @akegan How do you guys feel about changing the `application` argument name here to something like `formApiApplicationValues`, 'formValues', `formApplicationVals', etc.? Because ultimately `formApi.values` is what is getting passed to this function as the `application`. Renaming would help indicate this.

Reply:
sounds great to me! I am definitely easily confused with the different applications we're passing around right now

Reply:
sure we can do that. Just one comment. The shape is the same, so at the end of the day is an application. Naming the var formApi in the Page makes the page kind of "award" of the underline implementation. I feel the contract/spec of that method is that it will get an application shape regardless of the underline implementation.
however...not a big deal, sharing a thought.

Reply:
@fddayan has a very good point and that's the major downside to renaming the argument. However, due to the confusion we've been having with all the different applications being passed around, this change could help readability enough that it's worth the tradeoff. We'll try it and see how it goes.